test/Datadog.Trace.Tools.Analyzers.Tests/ThreadAbortAnalyzerTests.cs

```diff
@@ -0,0 +1,208 @@
+﻿// <copyright file="ThreadAbortAnalyzerTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
Are these test cases being invoked by our dotnet test commands?

Reply:
Yes, they're run as part of the various UnitTest stages. The fact they were (accidentally) MSTest vs XUnit doesn't affect that, but I've updated to XUnit now anyway 🙂